.travis.yml

```diff
@@ -21,7 +21,7 @@ before_install:
   - cd coalahtml/_coalahtml
   - bower install
   - cd -
-  - sudo pip3 install -r requirements.txt
+  - sudo pip3 install -U -r requirements.txt
```

Comment:
I'd suggest you do upgrade colorama in it's own line after this, Upgrading all the requirements will increase the build time frequently.
Ad it would be in an own line, you can add a comment explaining why you need to upgrade too :)